source/databricks/calculation-engine/package/constants/__init__.py

```diff
@@ -17,6 +17,6 @@
 from .partition_key_name import PartitionKeyName
 from .result_table_col_name import ResultTableColName
 
-DATABASE_NAME = "wholesale_output"
-RESULT_TABLE_NAME = "result"
+OUTPUT_DATABASE_NAME = "wholesale_output"
+ENERGY_RESULT_TABLE_NAME = "result"
```

Comment:
When doing this, should we also do this?
```suggestion
ENERGY_RESULT_TABLE_NAME = "energy_result"
```

Reply:
yes, I made a separate task on the story on this